### PR TITLE
Release 2.4.1

### PR DESCRIPTION
**Pull request by jonasscheid**
### `Added`
- Added low resolution settings (e.g. Iontrap) [#254](https://github.com/nf-core/mhcquant/pull/254) @steffenlem 

### `Fixed`

- Increased comet search, through altering the spectrum_batch_size from 500 to 0
- [#249](https://github.com/nf-core/mhcquant/pull/249) - nf-core template update (version 2.7.2) @marissaDubbelaar 
- [#258](https://github.com/nf-core/mhcquant/pull/258) - Adjusted decoy strategy to reverse [#255](https://github.com/nf-core/mhcquant/issues/255) and made consistent fdr-level flags [#228](https://github.com/nf-core/mhcquant/issues/228)
- [#845](https://github.com/nf-core/test-datasets/pull/845) - Adjusted nf-core test data set [#233](https://github.com/nf-core/mhcquant/issues/233)

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mhcquant/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/mhcquant _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/vcf_neoepitope_predictor.py
Comment (ramprasadn) [inline bin/vcf_neoepitope_predictor.py:1]: Make sure that you have the license information for all the python scripts in bin.. For example, https://github.com/nf-core/eager/blob/bb32ae3b0110b9a26b791f73a5324828d849271a/bin/endorS.py#L1-L4

File: modules/local/openms_decoydatabase.nf
Comment (ramprasadn) [inline modules/local/openms_decoydatabase.nf:1]: Is there a reason you have openms_decoydatabase as a local module? I have never used this tool, but it looks like it meets all the requirements to be a nf-core module :)

Comment (jonasscheid) [reply modules/local/openms_decoydatabase.nf:1]: The reason is actually that we did not have the time yet to make nf-core modules out of all the OpenMS tools. This is on the agenda once OpenMS 3.0.0 is released. I think that makes most sense imo

Comment (ramprasadn) [reply modules/local/openms_decoydatabase.nf:1]: Sure! 👍🏻

File: bin/vcf_neoepitope_predictor.py
Comment (jonasscheid) [reply bin/vcf_neoepitope_predictor.py:1]: https://github.com/nf-core/mhcquant/pull/262

Comment (jonasscheid) [reply bin/vcf_neoepitope_predictor.py:1]: @ramprasadn authorships and license updated :)

**Review by ramprasadn — APPROVED** [review]
LGTM! 👍🏻 🚀

File: nextflow.config [hunk 1]
```diff
@@ -219,12 +223,12 @@ dag {
...
-    author          = 'Leon Bichmann, Marissa Dubbelaar'
+    author          = """Leon Bichmann, Marissa Dubbelaar"""
     homePage        = 'https://github.com/nf-core/mhcquant'
-    description     = 'Identify and quantify peptides from mass spectrometry raw data'
+    description     = """Identify and quantify peptides from mass spectrometry raw data"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=21.10.3'
-    version         = '2.3.2dev'
+    nextflowVersion = '!>=22.10.1'
+    version = '2.4.1dev'
```
Comment (FriederikeHanssen) [inline nextflow.config:231]: dev needs to be removed before the release 🙈

**Comment by FriederikeHanssen** [comment]
Also just saw that in some places `exit` is still used. would be good to remove that and replace with `Nextflow.error` in the `.groovy` files and `error` in `.nf` files

**Pull request merged**